dataland-frontend/tests/e2e/specs/company-metadata/SearchCompaniesForFrameworkDataDropdownFilter.ts

```diff
@@ -17,7 +17,7 @@ before(function () {
 });
 
 describe("As a user, I expect the search functionality on the /companies page to adjust to the selected dropdown filters", () => {
-  it("LKSG and SFDR should be displayed in the framework dropdown even though they are not yet implemented", () => {
+  it("Lksg and Sfdr should be displayed in the framework dropdown even though they are not yet implemented", () => {
```

Comment:
I do not get the change here. This is a string and we do not need to follow camel case convention but can use the actual names for the frameworks (LkSG and SFDR)

Reply:
Repaired.